### PR TITLE
Poc/Store and keep wallet wallet state persistent

### PR DESCRIPTION
**Pull request by duc-q**
### Summary
Persisting extension state when User closes the extension. When locked, extension will ask for password for authentication.

### Checklist

-   [X ] I removed all commented or unnecessary code.
-   [X ] My code has covered these test cases (please list down your tested cases):

### CODE REVIEW

File: client/.env.development.local [hunk 1]
```diff
@@ -1,3 +1,3 @@
-REACT_APP_API_ROOT=http://localhost:3001
+REACT_APP_API_ROOT=https://testnet-api.casperdash.io
```
Comment (gaupoit) [inline client/.env.development.local:1]: hi please revert it. Let's use `yarn dev-extension-testnet` if you want connect to testnet API.

**Comment by gaupoit** [comment]
It works on my side. Thanks @duc-q . hi @djhahe please help to review and merge it. Cheers.

File: client/.env.development.local (hunk 1 above)
Comment (duc-q) [reply client/.env.development.local]: Updated

**Comment by djhahe** [comment]
@duc-q Please help to check if we need update permission in manifest.json to use chrome storage.
LGTM for now, in future we may need to consider to store state in memory as the article (https://auth0.com/blog/secure-browser-storage-the-facts/)

**Review by djhahe — APPROVED** [review]

**Comment by gaupoit** [comment]
hi a @duc-q , please double check the web app. There is an error when connecting to Casper Signer.
```
localStorage.js?4026:77 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'local')
    at eval (localStorage.js?4026:77:22)
    at new Promise (<anonymous>)
    at getChromeStorageLocal (localStorage.js?4026:76:12)
    at getConnectedAccountChromeLocalStorage (userActions.utils.js?e409:4:31)
    at eval (userActions.js?024a:72:71)
    at eval (index.js?b083:16:18)
    at eval (Heading.jsx?0f10:48:4)
    at invokePassiveEffectCreate (react-dom.development.js?61bb:23487:20)
    at HTMLUnknownElement.callCallback (react-dom.development.js?61bb:3945:14)
    at Object.invokeGuardedCallbackDev (react-dom.development.js?61bb:3994:16)
```

```
localStorage.js?4026:77 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'local')
    at eval (localStorage.js?4026:77:22)
    at new Promise (<anonymous>)
    at getChromeStorageLocal (localStorage.js?4026:76:12)
    at getConnectedAccountChromeLocalStorage (userActions.utils.js?e409:4:31)
    at eval (userActions.js?024a:72:71)
    at eval (index.js?b083:16:18)
    at eval (Heading.jsx?0f10:48:4)
    at invokePassiveEffectCreate (react-dom.development.js?61bb:23487:20)
    at HTMLUnknownElement.callCallback (react-dom.development.js?61bb:3945:14)
    at Object.invokeGuardedCallbackDev (react-dom.development.js?61bb:3994:16)
```

**Pull request merged**